### ISSUE TITLE
[Bug][a11y]: ColorLoupe svg should be hidden from screen readers

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3316]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

ColorLoupe

### Expected behavior

ColorLoupe should be hidden from screen readers.

### Actual behavior

Interacting with ColorArea using VoiceOver will sometime navigate to the svg image used within the ColorLoupe.

### What browsers are you seeing the problem in?

Chrome, Safari

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error
